components/pokemonapp.brs

```diff
@@ -60,3 +86,33 @@ function createPokemonSplashArt(splashArtObject as Object, selectedPokemon as dy
     m.top.appendChild(splashArtObject)
     splashArtObject.setFocus(true)
 end function
+
+function displayPokemonRating()
+    rating = CreateObject("roSGNode","StarRatingContentNode")
+    m.pokeRatingList.content = rating
+    observeRatingList()
```

Comment:
Why did you set the observer again? 
It was never [unobserved](https://developer.roku.com/docs/references/brightscript/interfaces/ifsgnodefield.md#unobservefieldfieldname-as-string-as-boolean)

Reply:
*It is the first time the RatingList observer is set.